pages/workflows/[id].js

```diff
@@ -0,0 +1,201 @@
+// node_modules
+import Link from "next/link";
+import PropTypes from "prop-types";
+// components
+import Attribution from "../../components/attribution";
+import Breadcrumbs from "../../components/breadcrumbs";
+import {
+  DataArea,
+  DataAreaTitle,
+  DataItemLabel,
+  DataItemValue,
+  DataItemValueUrl,
+  DataPanel,
+} from "../../components/data-area";
+import DbxrefList from "../../components/dbxref-list";
+import DocumentTable from "../../components/document-table";
+import { EditableItem } from "../../components/edit";
+import JsonDisplay from "../../components/json-display";
+import ObjectPageHeader from "../../components/object-page-header";
+import PagePreamble from "../../components/page-preamble";
+import SeparatedList from "../../components/separated-list";
+// lib
+import buildBreadcrumbs from "../../lib/breadcrumbs";
+import {
+  requestAnalysisSteps,
+  requestDocuments,
+} from "../../lib/common-requests";
+import errorObjectToProps from "../../lib/errors";
+import FetchRequest from "../../lib/fetch-request";
+import AliasList from "../../components/alias-list";
+import buildAttribution from "../../lib/attribution";
+import { isJsonFormat } from "../../lib/query-utils";
+
+export default function Workflow({
+  workflow,
+  analysissteps,
```

Comment:
You don’t have to change ths, but I was surprised you didn’t spell this as `analysisSteps`. Though for me it’s mostly noticeable because my source code spell checker feels this is a misspelling, but it would have said `analysisSteps` was spelled correctly.

Reply:
Decided to update this, I'd rather fix it now than forget about it.